engine/userguide/eng-image/multistage-build.md

```diff
@@ -0,0 +1,145 @@
+---
+description: Keeping your images small with multi-stage images
+keywords: images, containers, best practices
+title: Use multi-stage builds
+---
+
+{% include edge_only.md section="page" %}
+
+Multi-stage builds are a new feature in Docker 17.05, and they will be exciting
+to anyone who has struggled to optimize Dockerfiles while keeping them easy to
+read and maintain.
+
+> **Acknowledgment**:
+> Special thanks to Alex Ellis for granting permission to use his blog post
```

Comment:
@mstanleyjones can you include a link out for the name?

+> Special thanks to [Alex Ellis](https://twitter.com/alexellisuk) for granting permission to use his blog post